src/domUtils.js

```diff
@@ -1,4 +1,9 @@
-import {ensureArray} from "DEGJS/objectUtils";
+function ensureArray(obj) {
```

Comment:
I don't know if it's exactly a dom utility, but what do you think about just exposing this method in the returned object? If you decide that makes sense, please also add a quick line in the readme docs